src/Domain/Money.php

```diff
@@ -63,4 +63,27 @@ public function jsonSerialize(): mixed
     {
         return ['amountInPence' => $this->amountInPence, 'currency' => $this->currency->isoCode()];
     }
+
+    public function lessThan(Money $that): bool
+    {
+        if ($this->currency !== $that->currency) {
+            throw new \UnexpectedValueException("Cannot compare amounts with different currencies");
+        }
+
+        return $this->amountInPence < $that->amountInPence;
+    }
+
+    public function moreThan(Money $that): bool
+    {
+        if ($this->currency !== $that->currency) {
+            throw new \UnexpectedValueException("Cannot compare amounts with different currencies");
+        }
+
+        return $this->amountInPence > $that->amountInPence;
+    }
+
+    public function __toString()
+    {
+        return $this->amountInPence / 100 . $this->currency->isoCode();
```

Comment:
```suggestion
        return $this->currency->isoCode() . ' ' . ($this->amountInPence / 100);
```
I think this is more typical way to stringify when using the ISO codes?

Would need corresponding test tweaks.

Reply:
Yes I think you're right. I would avoid __toString for generating any user facing output, but will change to make any logs its used in a bit easier to read, then merge.